src/Components/Navbar/Navbar.scss

```
@@ -0,0 +1,117 @@
+.navigation {
+  position: relative;
+  .header {
+    display: flex;
+    justify-content: space-between;
+    height: 52px;
+    padding: 5px 0 0 9px;
+    background-color: white;
+  }
+
+  img {
+    height: 35px;
+    width: auto;
+  }
+
+  .navText {
+    display: inline-block;
+    padding: 12px 0 20px 0;
+    margin-left: 40px;
+    color: inherit;
+    background: linear-gradient(#000, #000) bottom / 0 0.1em no-repeat;
+    transition: 0.3s background-size;
+  }
+
+  .navText:hover {
+    background-size: 100% 0.2em;
+  }
+
+  ul {
+    display: flex;
+    list-style: none;
+    text-align: center;
+
+    li {
+      position: relative;
+      margin-left: 10px;
+    }
+
+    .dropBtn {
+      font-family: "Fira Sans", sans-serif;
+      font-size: 15px;
+      font-weight: 600;
+      padding: 0px 20px;
+    }
+  }
+
+  .matchBtn {
+    width: 119px;
+    height: 36px;
+    margin: -7px 5px 0 0;
+    padding: 7px 12px 6px 12px;
```

아래와 같이 작성해주실 수 있습니다 :) 
```suggestion
    padding: 7px 12px 6px;
```